ConsumerApi.Tests.Integration/Models/RegisterDeviceResponse.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Backbone.DevelopmentKit.Identity.ValueObjects;
+
+namespace Backbone.ConsumerApi.Tests.Integration.Models;
+
+public class RegisterDeviceResponse
+{
+    public DeviceId? Id { get; set; }
+    public string? Username { get; set; }
+    public DateTime CreatedAt { get; set; }
+    public DeviceId? CreatedByDevice { get; set; }
```

Comment:
Why is everything optional except for CreatedAt? Afaik none of these properties are ever null